include/picongpu/fields/incidentField/profiles/DispersivePulse.hpp

```diff
@@ -185,15 +185,16 @@ namespace picongpu
                                 + SPEED_OF_LIGHT * alpha * focusPos / (Unitless::W0 * Unitless::w);
 
                             // gaussian envelope in frequency domain
-                            float_X mag = math::exp(
-                                -(Omega - Unitless::w) * (Omega - Unitless::w) * Unitless::PULSE_DURATION
-                                * Unitless::PULSE_DURATION);
+                            float_X const freqExp = -(Omega - Unitless::w) * (Omega - Unitless::w)
+                                * Unitless::PULSE_DURATION * Unitless::PULSE_DURATION;
```

Comment:
One could rename it to envFreqExp, so that it would be in accordance with the following variable naming schemes.

Reply:
done